src/main/groovy/wooga/gradle/unity/UnityPlugin.groovy

```diff
@@ -148,6 +151,7 @@ class UnityPlugin implements Plugin<Project> {
             t.logToStdout.convention(t.logger.infoEnabled || t.logger.debugEnabled)
             t.toggleLogFile(true)
             t.buildTarget.convention(extension.defaultBuildTarget)
+            t.testFilter.convention(extension.testFilter)
```

Comment:
I think this should only be added to the `Test` tasks. See line 200

Reply:
Good point